pyrefly/lib/alt/function.rs

```diff
@@ -64,6 +64,18 @@ impl<'a, Ans: LookupAnswer> AnswersSolver<'a, Ans> {
         let skip_implementation = self.module_info().path().style() == ModuleStyle::Interface
             || class_metadata.is_some_and(|idx| self.get_idx(*idx).is_protocol());
         let def = self.get_idx(idx);
+        if def.metadata.flags.is_deprecated {
```

Comment:
hmm, doesn't this emit the error where the function is defined, not where it is called?

If the deprecated function exists and nobody calls it, there shouldn't be an error.